tmail-backend/jmap/extensions/pom.xml

```diff
@@ -57,6 +57,12 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-guice-configuration</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-common</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is not sorted correctly. It should come before `james-server-guice-configuration` as 'm' comes before 'n'.